CONTRIBUTING.md

```diff
@@ -93,6 +87,35 @@ on [Submitting a patch](#submitting-a-patch) for details.
 
 [Back to TOC](#table-of-contents)
 
+## Our pledge to the community
+
+Kong Gateway is not only an awesome open source project, but also part
+of the product offering of Kong Inc.  We have a large team of product
+people, software developers, testers and release engineers working on
+Kong Gateway.  We make many of the enhancements to Kong Gateway in the
+Community Edition, and thus they become available to the open source
+community even before our commercial customers get them.
+
+Recognizing that we operate as a commercial entity, we face the
+challenge of balancing our commercial interests with the desire to
+accommodate and support open source users and contributors
+effectively.
+
+In the Kong Gateway team, we're committed to maintaining a rapid and
+timely response to community contributions, promising to acknowledge
+and engage with them within a dedicated timeframe of 7 working
+days. However, it's important to note that while we strive to be as
+responsive as possible, we may not always be able to offer immediate
+solutions to every reported problem or incorporate every submitted
+pull request into the product.
+
+It also means that sometimes, we will be closing pull requests or
+issue reports when we made the decision that we will not be able to
+merge or resolve them in the foreseeable future.  We do that in the
```

Comment:
Are we going to have a hard standard to consider a ticket stale? I'm OK with either having discretionary power or not.

Reply:
I have added this:
> We automatically close issues and pull requests for which we do not get responses to our questions or update requests within 3 weeks.